src/main/java/spireCafe/interactables/patrons/dandadan/RightballPotion.java

```diff
@@ -0,0 +1,67 @@
+package spireCafe.interactables.patrons.dandadan;
+
+import com.badlogic.gdx.graphics.Color;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.helpers.PowerTip;
+import com.megacrit.cardcrawl.localization.PotionStrings;
+import com.megacrit.cardcrawl.potions.AbstractPotion;
+
+import spireCafe.Anniv7Mod;
+
+public class RightballPotion extends AbstractPotion {
```

Comment:
```suggestion
public class RightballPotion extends CustomPotion {
```

basemod.abstracts.CustomPotion doesn't do anything for now but we should always use the custom abstracts from basemod as base